cirq/circuits/qasm_output.py

```diff
@@ -127,8 +138,24 @@ def _decompose_(self, qubits: Sequence[ops.QubitId]) -> ops.OP_TREE:
 
     def __repr__(self) -> str:
         return 'cirq.QasmTwoQubitGate({}, {}, {}, {}, {}, {}, {})'.format(
-                self.before0, self.before1, self.x, self.y, self.z,
-                self.after0, self.after1)
+            self.before0, self.before1, self.x, self.y, self.z,
+            self.after0, self.after1)
+
+    def _unitary_(self) -> np.ndarray:
```

Comment:
This one got caught in the crossfire of https://github.com/quantumlib/Cirq/pull/1154 😨